src/components/Wizard/CreateVmWizard/CreateVmWizard.js

```diff
@@ -283,10 +283,23 @@ export class CreateVmWizard extends React.Component {
     }
   };
 
+  onHideWrapper = (...eventArgs) => {
+    this.wizardStepsNewVM.forEach(step => {
+      if (step.onCloseWizard) {
+        step.onCloseWizard(this.state.stepData[step.key].value, this.props.k8sKill);
```

Comment:
can you pls send k8sKill wrapped in an object so this can be easier extended for other props in future?

Reply:
done, I'm calling it `callerContext` to be aligned with my plan to wrap similar web-ui references in such an object and so pass them consistently through-out various web-ui-components calls.
Such a refactoring is planed for a follow-up.